lib/sm/api/message_drafts.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module SM
+  module API
+    module MessageDrafts
+      # post_create_message_draft: Creates a new draft, without attachments. If an id is included as
+      # a parameter, then the message draft is updated.
+      def post_create_message_draft(args = {})
+        json = perform(:post, 'message/draft', args.to_json, token_headers)
```

Comment:
I think the specs for this method must not be complete enough -- you're only testing "body" but if you tested additional attributes like recipient_id, you would have run into an issue with the strings not being tokenized. Just FYI